### ISSUE TITLE
0 items announced in tree view

### ISSUE THREAD

**Issue by justas2481** [microsoft/vscode#169493]
Type: <b>Bug</b>
@isidorn 
When I navigate a treview, screen reader reads all the items at first. When I do that for quite a long time (minute for example) it stops reading those items, though cursor still continues to move across items. In other words, items are still navigable with arrow keys, but screen reader doesn't read them anymore.
You can try this wit a debugger for example. Turn on NVDA screen reader, Write some code that generates an array of items and add a breakpoint in the code after array of items are generated. Then start debugging and go to that breakpoint. Then navigate to debug variables treview and expand that array of items, move arrows up/down for a minute to walk through them and you'll see that items aren't read by NVDA. It happens almost in every treview, not specifically with debugger. It wasn't happening with the previous versions.
I attach a screenshot so that issue is easier understood.
<img width="960" alt="treview_bug" src="https://user-images.githubusercontent.com/34254634/208293242-276c91ea-b53a-4701-b288-b69f817ab5c3.png">

VS Code version: Code 1.74.1 (1ad8d514439d5077d2b0b7ee64d2ce82a9308e5a, 2022-12-14T10:30:51.966Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No
Remote OS version: Linux x64 5.15.0-53-generic

**Comment by pleveris** [comment]
Can confirm that. Also I noticed this lag appears in the search results list, let's say, you wanna search for some code in all the project by CTRL+SHIFT+F, when navigating through the set of the results, after some time the same issue appears when screen reader somehow lost focus, but the cursor visually is moving along; To restore screen reader to the current focus we have to do some workarounds, we should jump out of the  VSCode window and get back to it again, then for some time it works as it should.
Sorry if I am not very specific here - just not sure how to handle this situation accurately as it seems it appears somehow randomly :)

**Comment by isidorn** [comment]
Thanks for reporting this issue. I wonder if it is related to https://github.com/microsoft/vscode/issues/169000
To verify it is not the same issue, can you try VS Code Insiders and let us know if you can still repro https://code.visualstudio.com/insiders/

Also a stable with a fix for the issue I mentioned should be available later this week.

**Comment by RichCaloggero** [comment]
I'm using the following version and treeviews seem to read ok, with one oddity: when expanding a branch, "0 items" is always announced, when in reality there are many items in the view. I was testing via the "open folder" dialog. 

Other than this issue, the treeview seems to read fine in this version, which has the fix for #169000.

Version: 1.75.0-insider (user setup)
Commit: e4bdaa1ea58f3f56cef36cd2d89d4ed1e5c9d443
Date: 2022-12-14T06:44:38.217Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19045
Sandboxed: Yes

**Comment by alexr00** [comment]
I can't reproduce this. If you still see it then we can reopen!
